src/main/java/org/broadinstitute/hellbender/tools/picard/sam/RevertSam.java

```diff
@@ -331,7 +330,7 @@ public void revertSamRecord(final SAMRecord rec) {
 
         if (REMOVE_ALIGNMENT_INFORMATION) {
             if (rec.getReadNegativeStrandFlag()) {
-                SAMRecordUtil.reverseComplement(rec);
+                rec.reverseComplement();
```

Comment:
In order to retain the same behavior here this should be rec.reverseComplement(true) so that it will make the changes in place instead of making a copy. Same thing in AbstractAlignmentMerger.